### ISSUE TITLE
Cant craft belly bound 

### ISSUE THREAD

**Issue by Jordanisgreat505** [Sheights/StarboundSimpleVoreMod#63]
Why can I see the bellybound in my c menu?

**Comment by AmoraSilver** [comment]
Because the full size store is not craftable or buyable, and the portable version you can only buy in the food hall.

**Comment by Jordanisgreat505** [comment]
Wait were is the food hall? is there any spawn codes? ive been looking for 2 hours all over the files

**Comment by AmoraSilver** [comment]
You can access the food hall from any ancient gate that you use to get to the outpost for the first time, including the one at the outpost itself.

**Comment by Jordanisgreat505** [comment]
Oh okay thanks for the help!

**Comment by windwakerfan** [comment]
i cant find any food hall ive spent days trying to find this bellybound store anywhere please help

**Comment by windwakerfan** [comment]
i know its installed because it shows up in the mod menu i have no idea how to fix this

**Comment by AmoraSilver** [comment]
http://i.imgur.com/SR97ZQz.png

You need to find one of these places on a starter world.  They are generally on a planet orbiting a gentle star, and have the lowest threat level and no ore greater than iron and gold.

**Comment by windwakerfan** [comment]
i have found that im post game and ive killed the tentacle planet dude

**Comment by AmoraSilver** [comment]
I'm guessing you updated to version 1.2 of Starbound?  I don't know what changes happens to the gates post game then since I don't have a character at that point anymore thanks to the Frackin' Universe mod no longer works and I had to delete everything, again (for the 15th time....)

Since this mod is not yet updated to version 1.2 for post game, the only option I can give you is a cheat way to get the mini store that has just about everything you need to get vore tenants and the vore objects.

Type these commands in this order:
/admin
/spawnitem smallbellyboundstore
/admin

That will give you the portable bellybound store.  Until I take the time to do a full playthrough again (or cheat), and/or the mod gets updated, that"s all I can do to help, sorry.

**Comment by windwakerfan** [comment]
yeah i started a new character and found it just fine i guess once you use the portal once you cant use them again because post game they all turn into those ancient vault things

**Comment by windwakerfan** [comment]
but how will i purchase the wares of the monster under the bathroom stalls

**Comment by windwakerfan** [comment]
( i totes didnt download this mod just to have access to more bones for my floran ship)

**Comment by windwakerfan** [comment]
also that spawn command didnt work

**Comment by AmoraSilver** [comment]
Oh oops, the item is actually case sensitive. I did not know that was possible, sorry.
 /spawnitem smallBellyBoundStore should work.
The item spawns wherever your mouse cursor is.

**Comment by windwakerfan** [comment]
cool that worked

**Comment by AmoraSilver** [comment]
http://issheights-true.tumblr.com/post/154655114232/hotfix-coming-soon

Just to let everyone know, Sheights knows of the post game issue and will most likely be fixed, plus the teleporter in the Food Hall will be able to be bookmarked so you don't have to hunt for an ancient portal every time you want to go there.

As a side note, I am only a fan of this mod and simply like to help in any way possible.  I do not know Sheights personally nor do I get anything from this other than the satisfaction of helping others.

**Comment by wyatt53** [comment]
Could I have some assistance? I tried to spawn the belly bound store using /spawnitem smallBellyBoundStore and all I get is Perfectly Generic item. Is the mod not installed correctly?

**Comment by AmoraSilver** [comment]
Either that or you did not spell it correctly.  Did you unzip the mod and have it in it's own folder under the mods folder?

**Comment by wyatt53** [comment]
I extracted the mod to it's own folder then moved the folder to "giraffe_storage/mods"
E:\Games\Steam\steamapps\common\Starbound\giraffe_storage\mods\StarboundSimpleVoreMod-8.4.0\StarboundSimpleVoreMod-8.4.0

**Comment by AmoraSilver** [comment]
Ah, I thought that was the issue.  You need to cut the folder you posted and paste it in the mod folder.

**Comment by wyatt53** [comment]
Sorry if I'm slightly confused. Cut and paste it into the mod folder under Starbound or remove the extra "StarboundSimpleVoreMod"?

**Comment by AmoraSilver** [comment]
Yeah, you need to remove the extra folder.

**Comment by wyatt53** [comment]
Odd, I did what you told me to and it still just spawns a generic item, a little green cube.
E:\Games\Steam\steamapps\common\Starbound\giraffe_storage\mods\StarboundSimpleVoreMod-8.4.0

**Comment by AmoraSilver** [comment]
You have to close and re-open the program first if you did not yet...

**Comment by wyatt53** [comment]
I did

**Comment by wyatt53** [comment]
Hmm, just checked and the mod doesn't show up in the mods folder in starbound.

**Comment by AmoraSilver** [comment]
Oh, I did not see it until now!  
You have it as:  
E:\Games\Steam\steamapps\common\Starbound\giraffe_storage\mods\StarboundSimpleVoreMod-8.4.0

It should be:
E:\Games\Steam\steamapps\common\Starbound\mods\StarboundSimpleVoreMod-8.4.0

**Comment by wyatt53** [comment]
The readme file told me to drop it into the giraffe_storage folder. Let me move the mod real quick.

**Comment by AmoraSilver** [comment]
The readme is outdated then, I'll message Sheights about that.

**Comment by wyatt53** [comment]
Alright, it's working now. Thank you btw.

**Comment by AmoraSilver** [comment]
You're welcome!

**Comment by wyatt53** [comment]
Now the only things I wish I could do is get to the Foodhall right when I start up the game. ^^;

**Comment by AmoraSilver** [comment]
You almost can now.  Once you get into the food hall the first time, go to the teleporter.  It now works correctly and lets you save it as a teleport location to travel from your ship!

**Comment by wyatt53** [comment]
That's good. Now I just have to find it for the first time.

**Comment by AmoraSilver** [comment]
You either go the the ancient gate on a gentle planet before beating the ruin, or use the teleporter at the 2-stop teleport shop.

**Comment by wyatt53** [comment]
Looked on the wiki and it says that a 2-stop teleport shop spawn at an outpost, which can only be accesed via ancient gate. Also 2 Stop Teleshop will not open until after a player on your account completes the The Erchius Mining Facility quest.

**Comment by AmoraSilver** [comment]
I'm guessing the Starbound wiki itself?  You do not need to complete that quest to make it to the food hall.

**Comment by wyatt53** [comment]
Oh, alright and yes, the wiki itself.

**Comment by AmoraSilver** [comment]
Alrighty then.  If you need any more help, just message here or open your own ticket.

**Comment by TommytheRabbit** [comment]
Hi, I hope I am not barging in, this is my first time doing this, if I need to make my own chat, I will. I just downloaded the more recent versions and added them to the mod folder and when I unzip it, my game crashes, when zipped nothing happens however, I have only just started the game. Do I need to keep it zipped and then complete a certain quest?

**Comment by thetechnodragon** [comment]
It needs to be unzipped to load, you'll also want a character extender like
xbawks

**Comment by thetechnodragon** [comment]
Once loaded you just need to get to the ark and use the 2 stop Tele shop
teleporter to reach the food hall, to my knowledge the bellybound store
isn't craftable anymore

**Comment by TommytheRabbit** [comment]
I unzip it but the game crashes...so far I am on the first mission, looking for core fragments and the teleporter only takes me to the beginning planet, so do I have to complete this too?

**Comment by thetechnodragon** [comment]
Core fragments activate the ancient gate and the ancient gate leads to the
ark, on the ark you will find a store called "two-stop tele-shop" that is
the teleporter you need

**Comment by thetechnodragon** [comment]
As for the crashes make sure the mod isn't in a folder in a folder in the
mod folder, the path to the _metadata file should be
/mods/simplestarboundvoremod/_Metadata

**Comment by TommytheRabbit** [comment]
Ok, I have unlocked the ark, for the file it is not in another file in the vore file.  I am also unsure which version to dowload since there is the 9.0.0 version but the most recent is the 8.6.1 version but I have come across something called 'StarboundSimpleVoreMod-master'

**Comment by TommytheRabbit** [comment]
I took the folder out of the zipped folder and removed the zipped version the game doesnt crash, I press C and no store...

**Comment by thetechnodragon** [comment]
You cannot craft the store, you must reach it via the teleporter on the ark

**Comment by TommytheRabbit** [comment]
Oh I see, there is this introductory video that has been saying that you must craft it, thank you for letting me know, I will try and find it now

**Comment by TommytheRabbit** [comment]
The only teleporter I can find is one that says outpost 2 teleshop and I cant go there for some reason

**Comment by thetechnodragon** [comment]
The teleporter is that 2 teleshop in its teleporter should be a location
called "food hall" if the mod is properly installed

**Comment by CC144Wartavian** [comment]
Question on what the name of the foodhall instance is, I can't find the teleport to it anywhere and I'm just trying to use the /warp command.

**Comment by florancetheavian** [comment]
to get to the foodhall, you must go to the outpost, then go to the 2stop teleshop teleporter. you should find it there. it will be labeled as "Foodhall- food vore mall"

**Comment by CC144Wartavian** [comment]
I've looked. Unable to find it. I need ot know what the instance of it so I can just warp to it. Seriously, I cannot see the teleport to it, so I need the warp ID instead.

**Comment by florancetheavian** [comment]
weird... it should be there if the mod was installed correctly. I'll look for a warp ID.

**Comment by CC144Wartavian** [comment]
Please do, thank you.

**Comment by florancetheavian** [comment]
okay, I am 97% sure I found it.
make sure you're in admin. if you installed the mod correctly this should work

/warp '{"instanceWorld": foodhall}'

I'll keep looking and try this out myself. make sure you type the cammand EXACTLY AS WRITTEN ABOVE. if you miss a capital letter it can create an error and cause the command not to work. I hope this works for you. remember warp commands can take time to load, and it takes longer than beaming.

**Comment by CC144Wartavian** [comment]
Yeah, I know how the warp command works... I just forgot that I had to use the instanceworld variable first. Thank you very much.

**Comment by florancetheavian** [comment]
anytime!

**Comment by CC144Wartavian** [comment]
...Starting to doubt myself, but I've double checked the installation of the mod and it's correctly installed, I'm 100% sure of it. I tried the command exactly as you sent it... And it didn't parse properly. Exact command, and the mod is very much installed right. It's just not working. Does it work for you?

**Comment by CC144Wartavian** [comment]
The command you sent was incorrect, a friend polished it up and it worked

**Comment by florancetheavian** [comment]
okay, well, glad you figured it out!

**Comment by No513TypePR** [comment]
I have a question, when I attempt to warp to the foodhall, it just tells me "let's try this warp again" I need the prey enabler, does anyone know the item id for it?
